forui/lib/src/foundation/tappable.dart

```diff
@@ -105,7 +111,25 @@ class _FTappableState extends State<FTappable> with SingleTickerProviderStateMix
           cursor: widget.enabled ? SystemMouseCursors.click : MouseCursor.defer,
           onEnter: (_) => setState(() => _hovered = true),
           onExit: (_) => setState(() => _hovered = false),
-          child: _child,
+          // We use a separate Listener instead of the GestureDetector in _child as GestureDetectors fight in
+          // "GestureArena" and only 1 GestureDetector will win. This is problematic if this tappable is wrapped in
+          // another GestureDetector as onTapDown and onTapUp might absorb EVERY gesture, including drags and pans.
+          child: Listener(
+            onPointerDown: (_) async {
```

Comment:
Do we really need to overcomplicate this though. Why not just set the Gesture Detector to be transparent?

Reply:
Do you mean translucent? Regardless of the hit test behavior, only a single gesture among several gesture detectors will win.